scripts/table.js

```diff
@@ -12,19 +15,58 @@ const validateEmail = (email) => {
     );
 };
 
+const sendToServer = (email, message) => {
+  const xhttp = new XMLHttpRequest();
+
+  xhttp.open("POST", url, true);
+
+  //Send the proper header information along with the request
+  xhttp.setRequestHeader("Content-Type", "application/x-www-form-urlencoded");
+  xhttp.send(`mail=${email}&feedback=${message}`);
+};
+
+const receieveFromServer = () => {
+  const xhttp = new XMLHttpRequest();
+
+  xhttp.open("GET", url, true);
+  xhttp.onreadystatechange = function () {
+    if (this.readyState == 4 && this.status == 200) {
+      var data;
+      //data.push(JSON.parse(xhttp.response));
+
+      data = JSON.parse(xhttp.responseText);
+
+      data.forEach((item, length) => {
+        console.log(i);
+        console.log(item);
+
+        var tbl = document.getElementById("myTable");
+        var row = tbl.insertRow(i);
+        var cell1 = row.insertCell(0);
+        var cell2 = row.insertCell(1);
+
+        cell1.innerHTML = item.mail;
+        cell2.innerHTML = item.feedback;
+        i++;
+      });
+    }
+  };
+
+  xhttp.send();
+};
+
 form.addEventListener("submit", (e) => {
+  var email = document.querySelector("#email").value;
+  var message = document.querySelector("#message").value;
   e.preventDefault();
-  if (!validateEmail(document.querySelector("#email").value)) {
+  if (!email) {
     alert("not a valid email format");
-  } else if (document.querySelector("#message").value === "") {
+  } else if (message === "") {
     alert("Write something");
   } else {
-    var tbl = document.getElementById("myTable");
-    var row = tbl.insertRow(i);
-    var cell1 = row.insertCell(0);
-    var cell2 = row.insertCell(1);
-    cell1.innerHTML = document.querySelector("#email").value;
-    cell2.innerHTML = document.querySelector("#message").value;
-    i++;
+    sendToServer(email, message);
```

Comment:
U need to implement promise. 

So, `receieveFromServer` is called  after the feedback is sent to server.

Also, request may fail. So, implement `try catch`